script/Cargo.toml

```diff
@@ -32,8 +32,8 @@ tokio = { version = "1.35.0", features = ["rt-multi-thread"] }
 
 [dev-dependencies]
 proptest = "1.0"
-ckb-db = { path = "../db", version = "= 0.114.0-pre" }
-ckb-store = { path = "../store", version = "= 0.114.0-pre" }
+ckb-db = { path = "../db", version = "= 0.114.0-pre", features = ["portable"] }
+ckb-store = { path = "../store", version = "= 0.114.0-pre", features = ["portable"] }
```

Comment:
why we need to add this.

Reply:
Because @XuJiandong is developed on an ancient machine, the code can only be compiled after adding this feature. It can be deleted in the future.